wandb/sdk/launch/launch_add.py

```diff
@@ -16,10 +16,16 @@
 
 
 def push_to_queue(
-    api: Api, queue_name: str, launch_spec: Dict[str, Any], project_queue: str
+    api: Api,
+    queue_name: str,
+    launch_spec: Dict[str, Any],
+    project_queue: str,
+    queue_entity: Optional[str] = None,
```

Comment:
`project_queue` and `queue_entity` .. can we make these consistent?

Reply:
yeah that was annoying me; I aligned with "queue_name". It would be funny if I ripped out all of Kyle's variable name choices while he was out though... ;) 
`project_queue` -> `queue_project` would be my personal preference....

Reply:
I also prefer `queue_*`

Reply:
probably OoS but when the time comes we need parallel change to get rid of this safely:

replace+deprecate (back compat): 

1. replace the `project_queue` keyword param with `queue_project`
2. add deprecated `project_queue` keyword param to end of paramlist

soft removal (back compat):
3. remove public docs

hard removal (not back compat; requires major)
4. remove support for old name

unclear if sdk team has back compat guidelines

Reply:
@gtarpenning sorry for the back and forth. lets hold off on this while we're under heightened time pressure.  it's not user-facing code anyway.